src/services/CertificateGenerationService.ts

```diff
@@ -62,10 +46,16 @@ class CertificateGenerationService {
             hgv_prs: config.documentNames.hgv_prs,
             trl_pass: config.documentNames.vtg5a,
             trl_fail: config.documentNames.vtg30,
-            trl_prs: config.documentNames.trl_prs
+            trl_prs: config.documentNames.trl_prs,
+            rwt: config.documentNames.rwt
         };
 
-        const vehicleTestRes: string = testResult.vehicleType + "_" + testType.testResult;
+        let vehicleTestRes: string;
+        if (CertificateGenerationService.isRoadworthinessTestType(testType.testTypeId)) { // CVSB-7677 is roadworthisness test
```

Comment:
![image](https://user-images.githubusercontent.com/50110006/70245691-6bc2ee80-176e-11ea-8bfa-241a74c4f909.png)
No test for this